Subject: use 64byte lvb len
From: Coly Li <coly.li@suse.de>

    This patch use 64 bytes lvb length for user space libdlm.

Signed-off-by: Coly Li <coly.li@suse.de>
---
Index: include/linux/dlm_device.h
===================================================================
--- a/include/linux/dlm_device.h
+++ b/include/linux/dlm_device.h
@@ -21,7 +21,7 @@
 #include <linux/dlm.h>
 #include <linux/types.h>
 
-#define DLM_USER_LVB_LEN	32
+#define DLM_USER_LVB_LEN	64
 
 /* Version of the device interface */
 #define DLM_DEVICE_VERSION_MAJOR 6
